healthcheck/settings/base.py

```diff
@@ -32,6 +32,14 @@
     "phonenumber_field",
     "celery",
     "django_filters",
+    # monitoring apps
+    "django_prometheus",
+    "health_check",
+    "health_check.db",
+    "health_check.cache",
+    "health_check.storage",
```

Comment:
I don't think we're using storage for anything, so we probably don't need to do a healthcheck for it.